.github/workflows/ci.yml

```diff
@@ -0,0 +1,322 @@
+      uses: actions/upload-artifact@v1.0.0
+      with:
+        name: linuxkit-${{matrix.arch}}
+        path: bin
+
+  build_packages:
+    name: Build Packages
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v1
+      with:
+        path: ./src/github.com/linuxkit/linuxkit
+
+    - name: Restore LinuxKit From Cache
+      uses: actions/cache@v1
+      with:
+        path: lkt
+        key: linuxkit-amd64-linux-${{hashFiles('src/**')}}
+
+    - name: Symlink Linuxkit
+      run: |
+        sudo ln -s `pwd`/lkt/linuxkit-amd64-linux /bin/linuxkit
+
+    - name: Build Packages
+      run: |
+        make -C pkg build
+
+  test_packages:
+    name: Packages Tests
+    needs: [ build_packages, build ]
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v1
+      with:
+        path: ./src/github.com/linuxkit/linuxkit
+    
+    - name: Install Pre-Requisites
+      run: |
+        export DEBIAN_FRONTEND=noninteractive
+        sudo apt-get update 
+        sudo apt-get install -qy qemu-utils qemu-system-x86 expect
+    
+    - name: Restore RTF From Cache
+      id: cache-rtf
+      uses: actions/cache@v1
+      with:
+        path: bin
+        key: rtf-${{hashFiles('Makefile')}}
+    
+    - name: Build RTF
+      if: steps.cache-rtf.outputs.cache-hit != 'true' 
+      run: make bin/rtf
+    
+    - name: Symlink RTF
+      run: |
+        sudo ln -s `pwd`/bin/rtf /usr/local/bin/rtf
+
+    - name: Restore LinuxKit From Cache
+      uses: actions/cache@v1
+      with:
+        path: lkt
+        key: linuxkit-amd64-linux-${{hashFiles('src/**')}}
+  
+    - name: Symlink Linuxkit
+      run: |
+        sudo ln -s `pwd`/lkt/linuxkit-amd64-linux /bin/linuxkit
+
+    - name: Run Tests
+      run: |
+          cd test
+          rtf -l build -v run -x linuxkit.packages
+
+  test_kernel:
+    name: Kernel Tests
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v1
+      with:
+        path: ./src/github.com/linuxkit/linuxkit
+   
+    - name: Install Pre-Requisites
+      run: |
+        export DEBIAN_FRONTEND=noninteractive
+        sudo apt-get update 
+        sudo apt-get install -qy qemu-utils qemu-system-x86 expect
+
+    - name: Restore RTF From Cache
+      id: cache-rtf
+      uses: actions/cache@v1
+      with:
+        path: bin
+        key: rtf-${{hashFiles('Makefile')}}
+    
+    - name: Build RTF
+      if: steps.cache-rtf.outputs.cache-hit != 'true' 
+      run: make bin/rtf
+    
+    - name: Symlink RTF
+      run: |
+        sudo ln -s `pwd`/bin/rtf /usr/local/bin/rtf
+
+    - name: Restore LinuxKit From Cache
+      uses: actions/cache@v1
+      with:
+        path: lkt
+        key: linuxkit-amd64-linux-${{hashFiles('src/**')}}
+
+    - name: Symlink Linuxkit
+      run: |
+        sudo ln -s `pwd`/lkt/linuxkit-amd64-linux /bin/linuxkit
```

Comment:
Is there a reason we link `linuxkit` to `/bin/linuxkit` but `rtf` to `/usr/local/bin/rtf`? Would be good to add a comment to the relevant commit if there is a good reason.

Reply:
No reason. I can fix this quickly before we merge.

Reply:
SGTM